src/pages/getting-started/app-guides/properties/index.mdx

```diff
@@ -12,7 +12,7 @@ desc: "Configure properties via user interface instead of calling the LiveChat C
 This document explains how to configure the Properties building block in the Developer Console. It was written for everyone aiming to configure properties in their integrations via user interface instead of calling the LiveChat Configuration API directly. The document assumes you're already familiar with the general idea of properties. If not, we encourage you to check the following resources:
 
 - <a href="https://developers.livechat.com/updates/livechat-properties/" target="_blank">Properties: what they are and how to configure them with LiveChat API v3.3+</a>
-- [Properties in the Configuration API](/management/configuration-api/#properties/)
+- [Properties in the Configuration API](/management/configuration-api/#properties)
```

Comment:
link with `/` at the end was directing to the top of the page, not directly to properties